albam/blender_ui/export_panel.py

```diff
@@ -261,17 +262,94 @@ def get_selected_item(context):
 
 @blender_registry.register_blender_type
 class ALBAM_OT_Pack(bpy.types.Operator):
+    FILEPATH = bpy.props.StringProperty(
+        name="File Path",
+        description="Filepath used for exporting the file",
+        maxlen=1024,
+        subtype='FILE_PATH',
+    )
+
     bl_idname = "albam.pack"
     bl_label = "Pack item"
+    filepath: FILEPATH
+
+    def invoke(self, context, event):  # pragma: no cover
+        vfs = context.scene.albam.file_explorer
```

Comment:
Using the imported list is pretty clever, I haven´t thought of it.
The idea I had was to select an existing arc file here from the real file system, and re-arrange the files inside it.
 
Not sure how user-friendly is, though, I still have to try it.